infrastructure/ecs-main/serverless.yml

```diff
@@ -156,6 +156,19 @@ custom:
     prd: "prd"
     dotcomstg: "preprd"
     dotcomprd: "prd"
+  site:
+    host:
+      dev: "docs-dev.mongodb.com" # NOTE: cannot be empty in redirect
+      stg: "mongodbcom-cdn.website.staging.corp.mongodb.com"
+      dotcomstg: "mongodbcom-cdn.website.staging.corp.mongodb.com"
+      prd: "www.mongodb.com"
+      dotcomprd: "www.mongodb.com"
+    prefix:
+      dev: "docs-dev"
```

Comment:
I think this should be just `docs` - that way we can have `docs-dev.mongodb.com/docs`

Reply:
I'll update this. Would we want to eventually do the same for staging? Move from `{stage-domain}/docs-qa` to `{stage-domain}/docs`?